### ISSUE TITLE
trinity compute nodes: no docker images 

### ISSUE THREAD

**Issue by themisa** [clustervision/trinity#371]
**Version:** r5_fixes f3f13d1
**Environment:** amaster

No docker images on compute nodes. Docker images fine on controller

```
[root@node001 ~]# docker images
FATA[0000] Get http:///var/run/docker.sock/v1.18/images/json: dial unix /var/run/docker.sock: no such file or directory. Are you trying to connect to a TLS-enabled daemon without TLS? 
[root@node001 ~]# docker ps
FATA[0000] Get http:///var/run/docker.sock/v1.18/containers/json: dial unix /var/run/docker.sock: no such file or directory. Are you trying to connect to a TLS-enabled daemon without TLS? 
```

**Comment by hansthen** [comment]
Table definition is wrong.

```
[root@controller(amaster) ~]# tabdump osimage
"centos7-x86_64-netboot-trinity",,"trinity","linux",,"netboot",,"centos7-x86_64",,,"Linux","centos7","x86_64",,,"cv_configure_prinic,cv_set_scheduler",,,,,

That is missing the cv_configure_storage postscript.
```

**Comment by themisa** [comment]
Fixed in **Version:** r5_fixes e5f4623
